src/components/space/SpaceHeader.js

```diff
@@ -93,10 +97,16 @@ class SpaceHeader extends Component {
   }
 
   renderDeleteButton() {
-    const { space } = this.props;
+    const {
+      space,
+      history: { goBack },
+    } = this.props;
     const { saved, id } = space;
     if (saved) {
-      return <DeleteButton spaceId={id} />;
+      const onSuccess = () => {
+        goBack();
```

Comment:
Are you sure `goBack` works for all cases?

Reply:
This only includes cases where the delete button in SpaceHeader is used. I cannot know where we will use it of course, but I can assume that the SpaceHeader is only used when viewing a space at `space/id`. Thus the goBack function will redirect to whatever page we come from (there are only two cases: Home page and Saved Spaces). Let me know if we should be more specific than `goBack`

Reply:
@pyphilia, we can leave it as is for now.